API.md

```diff
@@ -0,0 +1,97 @@
+# Basic API Guide
+
+## Login
+
+* Endpoint: `/login`
+* Verb: `POST`
+* Requires existing session: `false`
+* GET params: 
+* POST params:
+    - `email`: string
+    - `password`: string
+* Returns: 200 JSON string (JWT token)
+```json
+"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoiNWZjNTFkOWNhMGYzNTFmYjYyMjkwMDlhIiwidGllciI6MSwiZXhwIjoxNjA2ODQwNjAxLCJpYXQiOjE2MDY3NTQyMDEsImlzcyI6IlNreW5ldCBBY2NvdW50cyJ9.0dhkOMHFBcoZSSBDrVBca5SOSeU3zMEOLVfNQWf_cOI"
+```
+also 400, 401, 422, 500
```

Comment:
Should we document what these error codes mean?

Reply:
Yes, we should definitely do that but maybe a little later. We'll use Slate to document the API, just like in `siad`. I created this doc, so Karol could start building the front end. At first i thought it shouldn't even be under source control because it's temporary but then I decided to include it because we can just rework it into the slate format (I still need to look that up).

All in all, we don't need to worry about this doc too much for the moment.